src/lib.rs

```diff
@@ -138,10 +145,17 @@ async fn login(url: &Url) -> shvrpc::Result<(BoxedFrameReader, BoxedFrameWriter)
             let (reader, writer) = stream.split();
             let brd = BufReader::new(reader);
             let bwr = BufWriter::new(writer);
-            let frame_reader: BoxedFrameReader =
-                Box::new(SerialFrameReader::new(brd).with_crc_check(false));
-            let frame_writer: BoxedFrameWriter =
-                Box::new(SerialFrameWriter::new(bwr).with_crc_check(false));
+            let frame_reader: BoxedFrameReader = Box::new(SerialFrameReader::new(brd).with_crc_check(false));
+            let frame_writer: BoxedFrameWriter = Box::new(SerialFrameWriter::new(bwr).with_crc_check(false));
+            reset_session = true;
```

Comment:
Are you sure that `reset_session` should be `true` for `unixs`?

Reply:
Yes, he is.